java/server/src/org/openqa/selenium/grid/sessionqueue/NewSessionQueuer.java

```diff
@@ -76,6 +76,8 @@ protected NewSessionQueuer(Tracer tracer, Secret registrationSecret) {
         .with(requiresSecret),
       get("/se/grid/newsessionqueuer/queue/size")
         .to(() -> new GetNewSessionQueueSize(tracer, this)),
+      get("/se/grid/newsessionqueue")
```

Comment:
should it be `newsessionqueue` or `newsessionqueuer`?
In case we'd like to be consistent

Reply:
This was mainly in line with what it returns and the Http verb used.  "GET newsessionqueue" was the aim. But I now realize it is not consistent. I will fix it.